unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/ExploreV2/Scripts/ExploreV2Feature.cs

```diff
@@ -0,0 +1,76 @@
+using System;
+using UnityEngine;
+
+/// <summary>
+/// Main controller for the feature "Explore V2".
+/// </summary>
+public class ExploreV2Feature : PluginFeature
+{
+    public event Action OnOpen;
+    public event Action OnClose;
+
+    internal UserProfile ownUserProfile => UserProfile.GetOwnUserProfile();
+
+    internal IExploreV2MenuComponentView view;
+
+    public override void Initialize()
+    {
+        base.Initialize();
+
+        view = CreateView();
+        SetVisibility(false);
+
+        if (string.IsNullOrEmpty(ownUserProfile.userId))
+            ownUserProfile.OnUpdate += OnProfileUpdated;
+        else
+            OnProfileUpdated(ownUserProfile);
```

Comment:
I'm concerned that here we are using `ExploreV2Feature` as a UI controller. The `PluginFeature` should be the container of bridges and controllers, not the controller itself. By going to this direction, this logic should be in another class. The sole responsibility of the plugin should be to initialize/destroy its subsystems.

Reply:
You're right, it should be separated. Done in https://github.com/decentraland/unity-renderer/pull/1193/commits/faf2f1ae629988617cc13c6309b71f566815ac0c, thanks!